Util/ViolationFormatterInterface.php

```diff
@@ -11,29 +11,11 @@
 
 namespace FOS\RestBundle\Util;
 
-use FOS\RestBundle\Controller\Annotations\Param;
-use Symfony\Component\Validator\ConstraintViolationInterface;
-use Symfony\Component\Validator\ConstraintViolationListInterface;
+use FOS\RestBundle\Validator\ViolationFormatterInterface as BaseViolationFormatterInterface;
 
-interface ViolationFormatterInterface
+/**
+ * @deprecated since version 1.8, to be removed in 2.0. Use {@link BaseViolationFormatterInterface} instead.
```

Comment:
since this is going into master the deprecation notices should go into the 1.7 branch